sql/wsrep_trans_observer.h

```diff
@@ -394,6 +403,13 @@ static inline int wsrep_after_commit(THD* thd, bool all)
   {
     ret= thd->wsrep_cs().ordered_commit();
   }
+
+  WSREP_DEBUG("wsrep_after_commit: %d seqno=%lld for %s ret=%s",
```

Comment:
As above: Relevance vs. chance of merge conflicts.

Reply:
Removed.